api/dataset/satellite.go

```diff
@@ -82,20 +82,32 @@ type Point struct {
 // ToString writes out the bounding box to a string.
 func (b *BoundingBox) ToString() string {
 	coords := []string{
-		b.pointToString(b.LowerLeft),
-		b.pointToString(b.UpperLeft),
-		b.pointToString(b.UpperRight),
-		b.pointToString(b.LowerRight),
+		b.pointToString(b.LowerLeft, ","),
+		b.pointToString(b.UpperLeft, ","),
+		b.pointToString(b.UpperRight, ","),
+		b.pointToString(b.LowerRight, ","),
 	}
 	return fmt.Sprintf("{%s}", strings.Join(coords, ","))
 }
 
-func (b *BoundingBox) pointToString(point *Point) string {
+// ToGeometryString writes out the bounding box to a geometry string (POSTGIS).
+func (b *BoundingBox) ToGeometryString() string {
+	coords := []string{
+		b.pointToString(b.LowerLeft, " "),
+		b.pointToString(b.UpperLeft, " "),
+		b.pointToString(b.UpperRight, " "),
+		b.pointToString(b.LowerRight, " "),
+		b.pointToString(b.LowerLeft, " "),
+	}
+	return fmt.Sprintf("POLYGON((%s))", strings.Join(coords, ","))
+}
+
+func (b *BoundingBox) pointToString(point *Point, separator string) string {
```

Comment:
In case you hadn't run into this before - you could this a [Stringer](https://tour.golang.org/methods/17).